biigle/ptp
# Introduces PTP tab and Job

This MR introduces the first point to polygon functionalities, in particular:
- Introduces a Tab in the volume view for the conversion
![image](https://github.com/user-attachments/assets/4eb64a89-b15e-4500-8c33-01060834c33e)

- Creates a tab that can be used to create a PTP Job. 
![image](https://github.com/user-attachments/assets/3c8ebb9a-40e6-490c-9a61-6e394dc75304)

Once you have a Volume with point annotations, you can run a Point to Polygon job on a label with at least a point annotation. This will in turn run:
- A Job computing the Expected areas for the label
- A job uploading these expected areas
- A job running the conversion, using the median of these expected areas as the expected area for the job
- A job that uploads the new converted annotations

To test this MR:
- Use the `dev-modules-point-to-polygon` branch for the biigle root 
- Add PTP to the composer, for now using the VCS (If you think I should add it to Packagist let me know)
- Try to run the Point to Polygon conversion.
